src/types/timestamp.rs

```diff
@@ -572,6 +572,66 @@ impl Timestamp {
     pub fn precision(&self) -> Precision {
         self.precision
     }
+
+    /// Returns the year that has been specified in the [Timestamp].
+    pub fn year(&self) -> i32 {
+        self.date_time.year()
+    }
+
+    /// Returns the month that has been specified in the [Timestamp].
+    pub fn month(&self) -> u32 {
+        self.date_time.month()
+    }
+
+    /// Returns the day that has been specified in the [Timestamp].
+    pub fn day(&self) -> u32 {
+        self.date_time.day()
+    }
```

Comment:
For `year`, `month`, and `day`, I believe the current implementation always returns the UTC date. We'll need to offer local and `utc_` versions of each of those as well. (Consider 11PM on New Year's Eve in NYC, when it's already New Year's day in London.)

Reply:
added `to_utc` for `Timestamp`, removed all `utc_*` methods.

Reply:
`self.date_time.day()` is the UTC `day`. In order to return the local `day`, you have to localize the `year`/`month`/`day` fields of `self.date_time` to accommodate the new year's eve case I described above.

If the `Timestamp` doesn't have an offset because its precision is too low, you can use the current values.